Readability.js

```diff
@@ -1667,30 +1669,26 @@ Readability.prototype = {
 
         var linkDensity = this._getLinkDensity(node);
         var contentLength = this._getInnerText(node).length;
-        var toRemove = false;
 
         // Make an exception for elements with no p's and exactly 1 img.
         if (img > p && !this._hasAncestorTag(node, "figure")) {
-          toRemove = true;
+          return true;
```

Comment:
Alternatively, could leave this the same and just `return toRemove` at the end of the outer block.